src/main/java/com/simibubi/create/content/logistics/packager/PackagerBlockEntity.java

```diff
@@ -76,7 +78,7 @@ public class PackagerBlockEntity extends SmartBlockEntity {
 	public PackagerItemHandler inventory;
 	private final LazyOptional<IItemHandler> invProvider;
 
-	public static final int CYCLE = 20;
+	public static final int CYCLE = AllConfigs.server().logistics.packagePackCycle.get();
```

Comment:
This config value is only applied after the game is reloaded, when used this way.

Reply:
CYCLE is also used outside PackagerBlockEntity.java, which this PR doesn't account for.